tsconfig.json

```diff
@@ -18,15 +18,22 @@
 			"ES2020",
 			"ESNext"
 		],
-		"module": "ESNEXT",
-		"moduleResolution": "node",
+		"module": "preserve",
+		"moduleResolution": "bundler",
```

Comment:
This is similar to `nodenext` in that it obeys package.json `exports` except that it doesn't require extensions in imports.